kong/runloop/wasm.lua

```diff
@@ -18,6 +18,7 @@ local sha256 = utils.sha256_bin
 
 local VERSION_KEY = "wasm_filter_chains:version"
 local TTL_ZERO = { ttl = 0 }
+local ATTACH_OPTS = {}
```

Comment:
Due to gremlins, the wasm tests will not pass without specifically setting this isolation level.

This change is only quasi-related to the other build changes and will be revisited later on.